src/ai/defaultai.cc

```diff
@@ -269,6 +269,20 @@ void DefaultAI::think() {
 		}
 		return;
 	}
+	for (const Widelands::Game::PendingDiplomacyAction& pda : game().pending_diplomacy_actions()) {
+		if (pda.other == player_number()) {
+			// TODO(Nordfriese): The AI just makes a random choice every time.
+			// In the future, make a strategic decision here.
+			const bool accept = RNG::static_rand(1) == 0;
```

Comment:
```suggestion
			const bool accept = RNG::static_rand(2) == 0;
```

Reply:
maybe we should even use 3 or 4 or even 5 as base for the decision.
with 1 as base it ws nearly 100% chance to get accepted.